README.md

```diff
@@ -1 +1,69 @@
-# Leader_Board
\ No newline at end of file
+# Leader_board
+
+> This is a Leaderboard game project, where i used Leaderboard API documentation. Data is sent to the server by clicking submit button, and it is retrieved from the server by clicking refresh button.
+
+## Built With
+
+- HTML, CSS,Webpack & JS
+
+## Live Demo
+
+[Click to view Project]( https://kaweesi-matia.github.io/Leader_Board/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+HTML, CSS & JS
+
+### Setup
+
+Download the zip file for repository and unzip it.
+You should have a code editor
+
+### Install
+
+Install Linters and code editor
+
+### Usage
+
+N/A
+
+### Run tests
+
+To run tests you can use the following commands
+For HTML you should run " npx hint . " on your terminal in root directory of project
+For CSS you should run " npx stylelint "**/*.{css,scss}" " on your terminal in root directory of project
+For JavaScript you should run " npx eslint " on your terminal in root directory of project
+
+### Deployment
+
+N/A
```

Comment:
- [ ] Your live demo is not working. In order to make the README professional, kindly remove that topic or make the live demo work with Github pages.
 - [ ] The other sections are not clear and are not customized to this project. Please, refactor those sections so the user can understand how to install and run the application. In the "getting started" section you could write about how to clone the application, install the dependencies with `npm` and start the application. Since there are no tests(you will learn about tests with jest in future projects), you could change that section too.